fluent.ebnf

```diff
@@ -32,15 +32,17 @@ quoted-text          ::= ([^{"] | '\{' | '\"')+
 block-text           ::= NL __ '|' unquoted-pattern
 
 placeable            ::= '{' __ expression __ '}'
-expression           ::= quoted-pattern
+expression           ::= selector-expression | select-expression
+
+selector-expression  ::= quoted-pattern
                        | number
                        | identifier
                        | variable
-                       | select-expression
                        | member-expression
                        | call-expression
+                       | placeable
```

Comment:
Yes. It's not what the serialize would normally serialize to, but I don't think we need to artificially limit the syntax here. It doesn't change the meaning compared to just using `$foo`, just like `(1) === 1` in JavaScript.